cmd/root.go

```diff
@@ -35,10 +35,29 @@ var RootCmd = &cobra.Command{
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
 		out := cmd.OutOrStderr()
 		logrus.SetOutput(out)
+		verbosity, err := cmd.Flags().GetCount("verbose")
+		if err != nil {
+			return err
+		}
+		logrus.SetLevel(logLevel(verbosity))
```

Comment:
if we are going to have just two levels for the moment better use `verbose` as a boolean (defaulting to `false`)

Reply:
@andresmgot Why is it better to have a boolean?

Reply:
with this solution you need to use `--verbosity 0` or `--verbosity 1` (or `--verbosity 38924532` that will have the same result). I have never seen that usage, normally you just use `--verbosity` or `-v` if you want a detailed output

Reply:
As far as I know that is not how it would work. You would put -v and if you want more levels in the future, you would do -vvvv (i.e. for 4). But I might be wrong. @ngtuna can you confirm how this would work, please?

Reply:
ahh, I see, I though it was setting verbosity as a integer. Nevermind my comment then

Reply:
Well, I am not sure, that's why I am asking @ngtuna for confirmation :)

Reply:
Yes, it seems to work with -vvvv (or -v=4 as well): https://github.com/spf13/pflag/blob/master/count_test.go